src/main/java/net/fabricmc/installer/Handler.java

```diff
@@ -109,7 +112,7 @@ public JPanel makePanel(InstallerGui installerGui) {
 				loaderVersionComboBox.addItem(str);
 			}
 			loaderVersionComboBox.setSelectedIndex(0);
-			statusLabel.setText("Ready to install");
+			statusLabel.setText(installerGui.getBundle().getString("prompt.ready.install"));
```

Comment:
You seem to be using a few ways to get the bundle/translate.

installerGui.getBundle().getString()
translate()

is there a reason for this?

Reply:
I originally put the bundle as a field in the gui, but it turns out that oop design don't work well here